FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/mbedtls_config.h

```diff
@@ -121,6 +121,9 @@ int mbedtls_platform_recv( void * ctx,
                            unsigned char * buf,
                            size_t len );
 
+#define MBEDTLS_SSL_SEND mbedtls_platform_send
+#define MBEDTLS_SSL_RECV mbedtls_platform_recv
```

Comment:
Could we leave a comment as to what this function does?

Reply:
Sure, no prob.